core/src/main/java/org/testcontainers/containers/GenericContainer.java

```diff
@@ -278,10 +268,10 @@ public SELF dependsOn(Startable... startables) {
      * Note that the circular dependencies are not supported.
      *
      * @param startables a list of {@link Startable} to depend on
-     * @see Startables#deepStart(Collection)
+     * @see Startables#deepStart(Iterable)
      */
-    public SELF dependsOn(List<Startable> startables) {
-        dependencies.addAll(startables);
+    public SELF dependsOn(Iterable<? extends Startable> startables) {
```

Comment:
this will break binary compatibility

Reply:
Pushing a commit now to reinstate `dependsOn(List)`